src/dir.targets

```diff
@@ -174,6 +188,33 @@
     <Error Condition="'$(TestRunExitCode)' != '0'" Text="One or more tests failed while running tests from '$(MSBuildProjectName)' please check log for details!" />
   </Target>
 
+  <Target Name="CheckTestCategories"
+          AfterTargets="Test"
```

Comment:
That doesn't work as Condition is being calculated when Test target is finished and thus I can't change the condition in the target it depends on :( (looks like msbuild bug to me)